modules/overviews/app/components/project_custom_fields/sections/show_component.html.erb

```diff
@@ -1,37 +1,32 @@
 <%=
   component_wrapper do
-    flex_layout(border: :bottom, pb: 2, classes: 'op-project-custom-field-section-container', data: {
+    render(Primer::OpenProject::SidePanel::Section.new(
+      classes: 'op-project-custom-field-section-container',
       test_selector: "project-custom-field-section-#{@project_custom_field_section.id}"
-    }) do |details_container|
-      details_container.with_row(mb: 2) do
-        flex_layout(align_items: :center, justify_content: :space_between) do |heading|
-          heading.with_column(flex: 1) do
-            render(Primer::Beta::Heading.new(tag: :h4)) { @project_custom_field_section.name }
-          end
+    )) do |section|
+      section.with_title { @project_custom_field_section.name }
 
-          heading.with_column do
-            render(Primer::Beta::IconButton.new(
-              icon: :pencil,
-              tag: :a,
-              href: project_custom_field_section_dialog_path(project_id: @project.id, section_id: @project_custom_field_section.id),
-              data: {
-                controller: 'async-dialog'
-              },
-              scheme: :invisible,
-              button_icon: :pencil,
-              test_selector: "project-custom-field-section-edit-button",
-              aria: { label: I18n.t(:label_edit) },
-            ))
-          end if allowed_to_edit?
-        end
+      if allowed_to_edit?
+        section.with_action_icon(
+          icon: :pencil,
+          tag: :a,
+          href: project_custom_field_section_dialog_path(project_id: @project.id, section_id: @project_custom_field_section.id),
+          data: {
+            controller: 'async-dialog'
+          },
+          test_selector: "project-custom-field-section-edit-button",
+          aria: { label: I18n.t(:label_edit) }
+        )
       end
 
-      @project_custom_fields.each do |project_custom_field|
-        details_container.with_row(mb: 3) do
-          render(ProjectCustomFields::Sections::ProjectCustomFields::ShowComponent.new(
-            project_custom_field:,
-            project_custom_field_values: get_eager_loaded_project_custom_field_values_for(project_custom_field.id)
-          ))
+      flex_layout do |details_container|
+        @project_custom_fields.each do |project_custom_field|
+          details_container.with_row(mb: 3) do
```

Comment:
The `mb: 3` always adds additional space to the bottom which looks weird for the last element as there is a lot of space to the bottom